docs/apache-airflow-providers-amazon/transfer/dynamodb_to_s3.rst

```diff
@@ -25,7 +25,7 @@ to an existing Amazon Simple Storage Service (S3) bucket.
 Prerequisite Tasks
 ------------------
 
-.. include:: ../_partials/prerequisite_tasks.rst
+.. include:: ../operators/_partials/prerequisite_tasks.rst
```

Comment:
`prerequisite_tasks.rst` refers to some instructions/help to use all operators/sensors in amazon provider package. As such, we should move it to `docs/apache-airflow-providers-amazon/_partials`